common/containers/Tabs/GenerateWallet/components/EnterPassword.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import translate from 'translations';
+import { genNewKeystore } from 'libs/keystore';
+import PasswordInput from './PasswordInput';
+
+// VALIDATORS
+const minLength = min => value => {
```

Comment:
I hate to say I'm not familiar with this formatting, could you explain this line?